conf/cmi/core.base_field_override.paragraphs_library_item.paragraphs_library_item.paragraphs.yml

```diff
@@ -29,7 +29,7 @@ id: paragraphs_library_item.paragraphs_library_item.paragraphs
 field_name: paragraphs
 entity_type: paragraphs_library_item
 bundle: paragraphs_library_item
-label: Paragraphs
+label: Lohkot
```

Comment:
Hmm, voidaankohan me vaa vaihtaa näitä käännöksiä? Meinaan, että tuolla on noita autoupdateja ollu, jotka jumppaa toiseen suuntaan ja käsittääkseni cmi -kansiossa olevat pitäs ehkä olla enkuks?

Reply:
Pistetään takaisin, jostain automaatiosta nää tuli.